cli/bzk/service.go

```diff
@@ -90,26 +94,21 @@ func doRestartService() {
 		log.Fatal(err)
 	}
 
-	allContainers, err = client.Ps(&docker.PsOptions{
```

Comment:
You shouldn't have removed `allContainers`.

It acts as a sort of "cache" to avoid calling `docker ps -a` multiple time in the same bzk request

For instance, when calling `bzk service start`, PS will be called 3 times instead of one previously.

Maybe the naming is not right and this should be extracted in its own method, but removing this will have a big performance impact on hosts where the `docker ps -a` command can sometimes be very long